pydicom/fileutil.py

```diff
@@ -283,10 +283,6 @@ def path_from_pathlike(file_object):
         ``PathLike`` objects have been introduced in Python 3.6. In Python 3.5,
         only objects of type :class:`pathlib.Path` are considered.
```

Comment:
May want to remove the 3.5 part here. Also I think I wrote a 3.5-specific test, or at least excluded 3.5 from some tests.